pyproject.toml

```diff
@@ -166,54 +166,60 @@ select = [  # Base linting rule selections.
     "UP",  # Pyupgrade - note that some of are excluded below due to Python versions
     "YTT",  # flake8-2020: Misuse of `sys.version` and `sys.version_info`
     "ANN",  # Type annotations.
+    "ASYNC",  # Catching blocking calls in async functions
+    # flake8-bandit: security testing. https://docs.astral.sh/ruff/rules/#flake8-bandit-s
+    # https://bandit.readthedocs.io/en/latest/plugins/index.html#complete-test-plugin-listing
+    "S101", "S102",  # assert or exec
+    "S103", "S108",  # File permissions and tempfiles - use #noqa to silence when appropriate.
+    "S104",  # Network binds
+    "S105", "S106", "S107",  # Hardcoded passwords
+    "S110",  # try-except-pass (use contextlib.suppress instead)
+    "S113",  # Requests calls without timeouts
+    "S3",  # Serialising, deserialising, hashing, crypto, etc.
+    "S5",  # Unsafe cryptography or YAML loading.
+    "S602",  # Subprocess call with shell=true
+    "S701",  # jinja2 templates without autoescape
     "BLE",  # Do not catch blind exceptions
     "FBT",  # Disallow boolean positional arguments (make them keyword-only)
     "B0",  # Common mistakes and typos.
     "A",  # Shadowing built-ins.
+    "COM",  # Trailing commas
```

Comment:
This was something we previously had enabled in pylint that now has options in ruff.